### PR TITLE
test: extract so11y test and add logs

### PR DESCRIPTION
**Pull request by kezhenxu94**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
@hanahmily We found this in the logs, 

image

By logging in here

image

**Comment by wu-sheng** [comment]
We removed the so11y in the e2e for now. Please check and fix this ASAP. We need to locate the boundary.

File: test/e2e/e2e-test/src/test/java/org/apache/skywalking/e2e/SO11yE2E.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    protected HostAndPort swWebappHostPort;
+
+    @BeforeAll
+    void setUp() {
+        queryClient(swWebappHostPort);
+    }
+
+    @AfterAll
+    public void tearDown() {
+        trafficController.stop();
```
Comment (hanahmily) [inline test/e2e/e2e-test/src/test/java/org/apache/skywalking/e2e/SO11yE2E.java:100]: @kezhenxu94 this line throws NPE

**Comment by hanahmily** [comment]
After increasing the time range of several metrics, cases seem passed. But in the teardown phase, an NPE is thrown, needing @kezhenxu94 's help.

**Comment by wu-sheng** [comment]
Why time range matters? Could you explain a little?

**Comment by hanahmily** [comment]
> Why time range matters? Could you explain a little?

The key point is the average of the increase of relevant metrics. for example, in a single minute, the increase of level 1 aggregation is 1, 1, 0. The average of them is 0, less than 1.After increasing time range, we might get 2,2,1, the result of average is more than 1.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**